Author: Hans-Christoph Steiner <hans@eds.org>
Date:   Tue Jan 31 14:02:53 2012 -0500
Description: fix linux-gtk build
 fix gui/linux-gtk build on Debian/Ubuntu by using standard LDFLAGS & LIBS vars
Forwarded: yes, in the bug tracker

index f843293..19d713b 100644
--- a/gui/linux-gtk/Makefile
+++ b/gui/linux-gtk/Makefile
@@ -1,7 +1,8 @@
 CC ?=	gcc
 
 CFLAGS += -Wall `pkg-config --cflags gtk+-2.0`
-LFLAGS= `pkg-config --libs gtk+-2.0`
+LDFLAGS +=
+LIBS := `pkg-config --libs gtk+-2.0` -lm
 
 OBJS=	src/main.o src/interface.o src/callbacks.o src/ipc.o \
 	src/packet.o src/nodes.o src/routes.o
@@ -9,7 +10,7 @@ OBJS=	src/main.o src/interface.o src/callbacks.o src/ipc.o \
 all:	olsrd-gui
 
 olsrd-gui:	$(OBJS)
-	$(CC) $(LFLAGS) -o $@ $(OBJS)
+	$(CC) $(LDFLAGS) -o $@ $(OBJS) $(LIBS)
 
 clean:
 	rm -f $(OBJS)
